### PR TITLE
prow: Add more kubeflow repositories

### PR DESCRIPTION
**Pull request by gaocegege**
/cc @jlewi @YujiOshima 

We renamed the repo github.com/kubeflow/hp-tuning to github.com/kubeflow/katib, thus we should change the repo name here.

Signed-off-by: Ce Gao <gaoce@caicloud.io>

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -102,7 +102,7 @@ tide:
     - kubeflow/pytorch-operator
     - kubeflow/testing
     - kubeflow/tf-operator
-    - kubeflow/hp-tuning
+    - kubeflow/katib
```
Comment (jlewi) [inline prow/config.yaml]: Can you also add all the repositories that are currently missing

And 
kubeflow/experimental-kvc
kubeflow/experimental-seldon
kubeflow/experimental-beagle
kubeflow/caffe2-operator
kubeflow/reporting
kubeflow/website

Related to kubeflow/website#4

Comment (gaocegege) [reply prow/config.yaml]: OK, I will add it.

**Comment by gaocegege** [comment]
PTAL @jlewi

**Comment by jlewi** [comment]
Thanks
/lgtm

**Comment by BenTheElder** [comment]
testgrid config is in correct per the test failure. otherwise
/approve

**Comment by gaocegege** [comment]
Fixed the failure, sorry for that.

PTAL

**Pull request merged**